src/renderers/dom/client/__tests__/ReactBrowserEventEmitter-test.js

```diff
@@ -452,4 +452,18 @@ describe('ReactBrowserEventEmitter', function() {
     expect(idCallOrder[2]).toBe(getInternal(GRANDPARENT));
   });
 
+  it('should not crash ensureScrollValueMonitoring when createEvent returns null', function() {
+    var originalCreateEvent = document.createEvent;
+    document.createEvent = function() {
+      return null;
+    };
+    spyOn(document, 'createEvent');
+
+    var hasEventPageXY = ReactBrowserEventEmitter.testEventPageXY();
+    expect(document.createEvent.calls.count()).toBe(1);
+    expect(hasEventPageXY).toBe(false);
+
+    document.createEvent = originalCreateEvent;
```

Comment:
Could you please wrap expectations into `try / finally` so `document.createEvent` is always restored regardless of whether the test fails?